pkg/processor/k8sprocessor/README.md

```diff
@@ -9,87 +9,155 @@ in a cluster, keeps a record of their IP addresses and interesting metadata. Upo
+      pods:
+      - name: <pod_name_regex>
+
+    # See "Extracting metadata" documentation section below
+    extract:
+      # List of rules to extract pod annotations into attributes.
+      # See the "Field extract config" documentation section below for details on how to use it.
+      # By default, no pod annotations are extracted into attributes.
+      # default: []
+      annotations:
+      - key: "*"
+        tag_name: k8s.pod.annotation.%s
+
+      # If a pod is associated with more than one service, delimiter will be used to join the service names.
+      # default: ", "
+      delimiter: <delimiter>
+
+      # List of rules to extract pod labels into attributes.
+      # See the "Field extract config" documentation section below for details on how to use it.
+      # By default, no pod labels are extracted into attributes.
+      # default: []
+      labels:
+      - key: "*"
+        tag_name: k8s.pod.label.%s
+
+      # List of pod metadata to extract into attributes.
+      # See "Extracting metadata" documentation section below for details.
+      # default: []
+      metadata:
+      - clusterName
+      - containerId
+      - containerImage
+      - containerName
+      - cronJobName 
+      - daemonSetName
+      - deploymentName
+      - hostName
+      - jobName
+      - namespace
+      - nodeName
+      - podId
+      - podName
+      - replicaSetName
+      - serviceName
+      - startTime
+      - statefulSetName
+
+      # List of rules to extract namespace labels into attributes.
+      # See the "Field extract config" documentation section below for details on how to use it.
+      # By default, no namespace labels are extracted into attributes.
+      # default: []
+      namespace_labels:
+      - key: "*"
+        tag_name: k8s.namespace.label.%s
+        
+      # Specifies the names of the attributes to put the extracted metadata in.
+      # See "Extracting metadata" documentation section below for details.
+      # For example, if `deploymentName` exists in the `extract.metadata` list,
+      # the name of the deployment will be put by default in an attribute named `k8s.deployment.name`.
+      # The following map defines the defaults.
+      # To override any of the defaults, specify a different attribute name for a selected key.
+      tags:
+        clusterName: k8s.cluster.name
+        containerID: k8s.container.id
+        containerImage: k8s.container.image
+        containerName: k8s.container.name
+        cronJobName: k8s.cronjob.name
+        daemonSetName: k8s.daemonset.name
+        deploymentName: k8s.deployment.name
+        hostName: k8s.pod.hostname
+        jobName: k8s.job.name
+        namespaceName: k8s.namespace.name
+        nodeName: k8s.node.name
+        podID: k8s.pod.id
+        podName: k8s.pod.name
+        replicaSetName: k8s.replicaset.name
+        serviceName: k8s.service.name
+        statefulSetName: k8s.statefulset.name
+        startTime: k8s.pod.startTime
+
```

Comment:
btw, we should use `resourceprocessor` to handle this feature

Reply:
💯 agree